src/sanitizer.js

```diff
@@ -251,7 +266,11 @@ export function sanitizeHtml(html) {
           continue;
         }
         emit(' ');
-        emit(attrName);
+        if (bindAttribsIndices.hasOwnProperty(i)) {
```

Comment:
If we [bind attribute name sanitization](https://github.com/ampproject/amphtml/pull/7602/files#diff-953786501fb82b0101b7d15a1b3873cbR196) after [`savedAttribs `](https://github.com/ampproject/amphtml/pull/7602/files#diff-953786501fb82b0101b7d15a1b3873cbR208), we can pull the name directly from `savedAttribs` without checking if it's a bind attribute.